Quaternion.mpl

```diff
@@ -192,3 +195,28 @@ axisAngleQuaternion: [
     ah cos
   ) quaternion
 ];
+
+rotationQuaternion: [
+  rotation:;
+
+  rotation 1.0e-12 rotationQuaternion
+];
+
+rotationQuaternion: [rotation: threshold:;; TRUE] [
```

Comment:
This overload checks only stack depth and will break if user wants to use 1-arg version when he has more values on uncontrolled stack. I suggest checking that `threshold` is a floating point number.